### PR TITLE
init fvm styles

### PR DESCRIPTION
**Pull request by t0rbik**
### Focus of this PR:
Update the frontend design and layout.

### Detailed summary:
- Updated logo images
- Updated header and mobileHeader components
- Updated typography styles in various pages
- Updated styles of ssBribes component
- Updated styles of transactionQueue component
- Updated styles of ssVestsTable component
- Updated styles of Vesting page
- Updated styles of ConnectButton component
- Updated styles of options/stake component
- Updated styles of options/reward component
- Updated styles of ssVest/TransferNFT component
- Updated styles of ssVotes component
- Updated styles of options/convert component
- Updated styles of ssVest/MergeNFT component
- Updated styles of pages/configure/index page
- Updated styles of common/PageWrapper component
- Updated tailwind.config.js file

> The following files were skipped due to too many changes: `Frontend-v1-Original/components/common/massiveInput.tsx`, `Frontend-v1-Original/components/ssVests/partnersVests.tsx`, `Frontend-v1-Original/pages/home/index.tsx`, `Frontend-v1-Original/components/liquidityPairs/LiquidityPairsTable.tsx`, `Frontend-v1-Original/components/layout/layout.tsx`, `Frontend-v1-Original/components/ssRewards/ssRewards.tsx`, `Frontend-v1-Original/components/ssBribeCreate/ssBribeCreate.tsx`, `Frontend-v1-Original/components/swap/swap.tsx`, `Frontend-v1-Original/components/navigation/navigation.tsx`, `Frontend-v1-Original/components/liquidityManage/LiquidityCreate.tsx`, `Frontend-v1-Original/components/options/redeem.tsx`, `Frontend-v1-Original/components/liquidityManage/LiquidityManage.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: Frontend-v1-Original/tailwind.config.js [hunk 1]
```diff
@@ -43,22 +40,25 @@ module.exports = {
...
-        secondaryGray: "#7e99b0",
+        primary: "#00E8CA",
         secondary: "#7e99b0",
-        primaryBg: "#272826",
+        primaryBg: "#232323",
         deepBlue: "#212b48",
         deepPurple: "#040105",
         accent: "#ffda58",
         success: "#07bc0c",
         warning: "#ffda58",
```
Comment (oxSaturn) [inline Frontend-v1-Original/tailwind.config.js:50]: Happened to see this on twitter, and would love to share this thread https://twitter.com/ryanflorence/status/1673847474941997056, basically why personally i would avoid using any semantic css classes those days when possible.

File: Frontend-v1-Original/components/layout/layout.tsx [hunk 2]
```diff
@@ -30,10 +28,9 @@ export default function Layout({
         <meta name="og:title" content="FVM" />
         <meta name="twitter:card" content="summary_large_image" />
       </Head>
-      <div className={classes.background} />
-      <div className={classes.greyGlow} />
-      <div className={classes.greenGlow} />
-      <div className={classes.content}>
+      <div className="pointer-events-none absolute top-0 right-0 -z-10 h-full w-full bg-appBg" />
+      <div className="pointer-events-none absolute top-0 right-0 -z-10 h-full w-full bg-waves bg-contain bg-no-repeat" />
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/layout/layout.tsx:32]: Maybe hide the background image on mobile, this image is 2.1MB, not small, would be great to replace it with a svg. And I reckon we can set a `opacity` like `0.3` to the background layer, currently it's a little disturbing to me:

image

**Review by oxSaturn — COMMENTED** [review]
Color looks good, some ideas about the background image.

**Comment by oxSaturn** [comment]
I'll fix the failed unit test action once I figure out what's going on with this issue https://github.com/wagmi-dev/references/issues/370 I ran into locally.

File: Frontend-v1-Original/tailwind.config.js (hunk 1 above)
Comment (t0rbik) [reply Frontend-v1-Original/tailwind.config.js:50]: Yeah i ve seen it. Didnt quite catch the take with colour names haha. Are those semantic do you reckon?

Comment (oxSaturn) [reply Frontend-v1-Original/tailwind.config.js:50]: They are. Chances are you might have `deeperBlue` and `deepestBlue` or `lighterBlue` in future. Mostly harmless when you code them first time, but could be headache when you want to extend or refactor, just reminds me of those days using BEM naming convention.

File: Frontend-v1-Original/tailwind.config.js [hunk 3]
```diff
@@ -43,12 +40,10 @@ module.exports = {
...
       colors: {
-        primary: "#06fc99",
-        secondaryGray: "#7e99b0",
+        primary: "#00E8CA",
         secondary: "#7e99b0",
-        primaryBg: "#272826",
-        deepBlue: "#212b48",
-        deepPurple: "#040105",
+        background: "#232323",
+        appBackground: "#021716",
```
Comment (oxSaturn) [inline Frontend-v1-Original/tailwind.config.js:46]: Those would still be semantic IMO.

Say I want to reuse this `#232323` color somewhere else other than background usage, I would either:

1. define another same color with different keyword in tailwind.config.js
2. or just reuse the `background` keyword, like `border-background` - which would be wrong

That semantic in `background` is limiting us from reuse.

I would just define something maybe like:

```
{
  black: {
    700: '#232323
  }
}
```
Btw, reckon we don't have to refactor those now, can do it later once we figure out the new color palettes.

**Pull request merged**